core/domain/collection_services.py

```diff
@@ -1243,44 +1240,13 @@ def get_next_page_of_all_non_private_commits(
     ) for entry in results], new_urlsafe_start_cursor, more)
 
 
-def _should_index(collection):
-    """Checks if a particular collection should be indexed.
-
-    Args:
-        collection: Collection.
-    """
-    rights = rights_manager.get_collection_rights(collection.id)
-    return rights.status != rights_manager.ACTIVITY_STATUS_PRIVATE
-
-
-def _collection_to_search_dict(collection):
-    """Converts a collection domain object to a search dict.
-
-    Args:
-        collection: Collection. The collection domain object to be converted.
-
-    Returns:
-        The search dict of the collection domain object.
-    """
-    doc = {
-        'id': collection.id,
-        'title': collection.title,
-        'category': collection.category,
-        'objective': collection.objective,
-        'language_code': collection.language_code,
-        'tags': collection.tags,
-        'rank': _DEFAULT_RANK,
-    }
-    return doc
-
-
 def clear_search_index():
     """Clears the search index.
 
     WARNING: This runs in-request, and may therefore fail if there are too
     many entries in the index.
     """
-    search_services.clear_index(SEARCH_INDEX_COLLECTIONS)
+    gae_search_services.clear_index(SEARCH_INDEX_COLLECTIONS)
```

Comment:
Oh, this is interesting. How feasible do you think it would be to move the functions which require gae_search_services to search_services, so that the latter is the only thing interfacing with gae_search_services?

Reply:
Done.